guides/common/modules/con_transport-modes-for-remote-execution.adoc

```diff
@@ -0,0 +1,28 @@
+[id="transport-modes-for-remote-execution_{context}"]
+= Transport Modes for Remote Execution
+
+You can configure your {Project} to use two different modes of transport for remote job execution.
+
+On {SmartProxies} in `ssh` mode, remote execution uses the SSH service to transport job details.
+This is the default transport mode.
+The SSH service must be enabled and active on the target hosts.
+The remote execution {SmartProxy} must have access to the SSH port on the target hosts.
+Unless you have a different setting, the standard SSH port is 22.
+
+On {SmartProxies} in `pull-mqtt` mode, remote execution uses Message Queueing Telemetry Transport (MQTT) to publish jobs it receives from {ProjectServer}.
+The host subscribes to the MQTT broker on {SmartProxy} for job notifications using the `yggdrasil` pull client.
+After the host receives a notification, it pulls job details from {SmartProxy} over HTTPS, runs the job, and reports results back to {SmartProxy}.
+
+To use the `mqtt-pull` mode, you must enable it on {SmartProxyServer} and on the target hosts.
```

Comment:
On hosts, pull is not technically a "mode." It's just that the pull client must be running and subscribed. How about
```suggestion
To use the `mqtt-pull` mode, you must enable it on {SmartProxyServer} and configure the pull client on the target hosts.
```